test/lib/config/index.js

```
@@ -0,0 +1,703 @@
+                            browsers: {
+                                b1: mkBrowser_(_.set({}, option, 15.5))
+                            }
+                        });
+
+                        ConfigReader.prototype.getConfigFromFile.returns(readConfig);
+
+                        const config = Config.create({});
+
+                        assert.throws(() => config.parse(), Error, 'Field must be an integer number');
+                    });
+                });
+
+                it(`should set ${option} option to all browsers`, () => {
+                    const readConfig = mkConfig_({
+                        browsers: {
+                            b1: mkBrowser_(),
+                            b2: mkBrowser_()
+                        }
+                    });
+
+                    ConfigReader.prototype.getConfigFromFile.returns(readConfig);
+
+                    const parsedConfig = Config.create({}).parse();
+
+                    assert.isDefined(defaults[option]);
+                    assert.equal(parsedConfig.browsers.b1[option], defaults[option]);
+                    assert.equal(parsedConfig.browsers.b2[option], defaults[option]);
+                });
+
+                it(`should override ${option} option`, () => {
+                    const readConfig = mkConfig_({
+                        browsers: {
+                            b1: mkBrowser_(),
+                            b2: mkBrowser_(_.set({}, option, 13))
+                        }
+                    });
+
+                    ConfigReader.prototype.getConfigFromFile.returns(readConfig);
+
+                    const parsedConfig = Config.create({}).parse();
+
+                    assert.equal(parsedConfig.browsers.b1[option], defaults[option]);
+                    assert.equal(parsedConfig.browsers.b2[option], 13);
+                });
+            });
+        });
+
+        describe('retry', () => {
+            describe('should throw error if retry', () => {
```

По-моему, можно и не выносить это в отдельный `describe`.
`should throw an error if retry is not a number` - вроде норм читается.